### ISSUE TITLE
[icetray] I3Module.h duplicate declarations of I3Frame (Trac #1946)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1938]
`I3Module.h` includes `I3Frame.h` and also forward declares class I3Frame.  You really don't need to do both.

Also, if you're going to include `I3Frame.h`, just use the `FrameFifo` declaration from there instead of declaring your own.

Migrated from https://code.icecube.wisc.edu/ticket/1946
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:52", 
    "description": "`I3Module.h` includes `I3Frame.h` and also forward declares class I3Frame.  You really don't need to do both.\n\nAlso, if you're going to include `I3Frame.h`, just use the `FrameFifo` declaration from there instead of declaring your own.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067232264679", 
    "component": "combo core", 
    "summary": "[icetray] I3Module.h duplicate declarations of I3Frame", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2017-02-03T21:13:59", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170420T18:04:54: 
* olivas commented:

> In changeset 2856:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2856"
> minor cleanup.  closes #1938.
> ```


* olivas changed status from "new" to "closed"
* olivas changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T20:55:24: nwhitehorn commented:

> In changeset 168803:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168803"
> minor cleanup.  closes #1938.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:52: 
* olivas changed milestone from "April 2017" to ""
* olivas commented:

> Milestone April 2017 deleted
